webpack.config.js

```diff
@@ -1,10 +1,16 @@
 const path = require('path')
+const ForkTsCheckerWebpackPlugin = require('fork-ts-checker-webpack-plugin')
 
 const libraryName = 'Leanplum'
 
 class DtsBundlePlugin {
   apply(compiler) {
-    compiler.plugin('done', function () {
+    compiler.hooks.done.tap('DtsBundlePlugin', (stats) => {
+      // Do not bundle TypeScript declaration files if there are errors.
+      if (stats.compilation.errors && stats.compilation.errors.length) {
+        return
```

Comment:
Using process.exit(1) here should cause the build to fail properly.

Reply:
Thanks for the suggestion!

I tried it and it fails the build (which is the correct behavior), however the errors from the TypeScript compiler are not displayed in this case, only this:
```
error Command failed with exit code 1.
```

I guess that there should be a better way to do this but it'll require some additional research.